downstream/modules/aap-hardening/proc-configure-centralized-logging.adoc

```diff
@@ -31,7 +31,7 @@ awx-manage print_settings LOG_AGGREGATOR_MAX_DISK_USAGE_PATH
 If this field is not set to "/var/log/awx", then follow these configuration steps: 
 +
 --
-.. Open a web browser and navigate to: link:https://<Automation Controller server>/api/v2/settings/logging/ (If the "Log In" button is displayed, click it, enter an Automation Controller administrator's credentials, and continue).
+.. Open a web browser and navigate to: link:https://<{ControllerName} server>/api/v2/settings/logging/ (If the "Log In" button is displayed, click it, enter an Automation Controller administrator's credentials, and continue).
```

Comment:
i'd change this to "Open a web browser and navigate to https://<FQDN>/api/v2/settings/logging/, where <FQDN> is the fully-qualified hostname of your automation controller.  If the "Log In" button is displayed, click it, log in as an automation controller adminstrator account, and continue."

Reply:
Done.